cmd/dcrctl/config.go

```diff
@@ -178,13 +178,20 @@ func loadConfig() (*config, []string, error) {
 		RPCCert:    defaultRPCCertFile,
 	}
 
+	// Create the home directory if it doesn't already exist.
+	err := os.MkdirAll(btcdHomeDir, 0700)
```

Comment:
also I thought we agreed to remove all of the dcrd directory creation from dcrctl.  if any directories/files are created by this tool, they should be in ~/.dcrrctl